examples/macro.rs

```diff
@@ -34,20 +34,18 @@ mod store_rpc {
     #[derive(Debug, Serialize, Deserialize)]
     pub struct ConvertFileResponse(pub Vec<u8>);
 
-    use super::Store;
     derive_rpc_service! {
```

Comment:
Not sure about naming. This does not really derive anything. It's not a derive macro, and it also does not really look at types in detail.

So maybe call it `define_rpc_service` ?

Reply:
what about just `rpc_service!`?

Reply:
`rpc_service` works for me. Just don't want `derive` in it.